### PR TITLE
Minor text fixes

### PR DESCRIPTION
**Pull request by arsenyjin**
I've went through the docs and everything did work for me, but a couple of inaccuracies in the text created a little bit of frictions. Hope this editions will help future generations!

### CODE REVIEW

File: Workflow.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ npx aragon ipfs
 ```
 
 
-Then go to the aragon client project and change the `environment.js` file to the following: https://gist.github.com/xseignard/e1952e53e62faa52b4f625344f5fba07
+Then go to the aragon client project and create the `environment.js` file to the following: https://gist.github.com/xseignard/e1952e53e62faa52b4f625344f5fba07
```
Comment (deamme) [inline Workflow.md]: We have a better PR for this part. I think it still should be change since we are modifying the `environment.js` in `src` folder.

Comment (arsenyjin) [reply Workflow.md]: I actually had no `environment.js` file in the `aragon` folder

Comment (deamme) [reply Workflow.md]: And it still worked? @xseignard

Comment (xseignard) [reply Workflow.md]: https://github.com/aragon/aragon/blob/master/src/environment.js

**Pull request closed without merging**